LondonBritishLibrary/orient/BLorient764.xml

```diff
@@ -53,67 +479,102 @@ schematypens="http://relaxng.org/ns/structure/1.0"
-                                
-                                
+                                <foliation>One unnumbered blank leaf at the end.</foliation>
+                                <condition key="good"><locus target="#89">F. 89</locus> and <locus target="#102">102</locus> have been
+                              cut and are of smaller size than the rest, but there is no loss of text.</condition>
+                                      
+
                             </supportDesc>
-                            
+
                             <layoutDesc>
-                                <layout columns="3" writtenLines="22" cert="high"/>
+                                <layout columns="2" writtenLines="22" cert="high"/>
                             </layoutDesc>
                         </objectDesc>
-                        
+
                         <handDesc>
-                            
+
                             <handNote script="Ethiopic" xml:id="h1">
-                                <desc>The manuscript is written in a fine hand, of the <date> 17th century</date>. 
-                                    
+                                <desc>The manuscript is written in a fine hand, of the <date> 17th century</date>.
                                 </desc>
-                                
                                 <date notBefore="1600" notAfter="1700" resp="PRS10171Wright"/>
-                                
                             </handNote>
-                            
-                        </handDesc>   
-                        <additions>
-                            
-                            
-                        </additions>
-                        
+                        </handDesc>
+
+<additions>
+<list>
+
+  <item xml:id="a1">
+  <locus target="#3r"></locus>
+    <desc type="OwnershipNote">The note states that the manuscript belongs to the church of
+            <placeName ref="INS0101MadhaneAlam"/>.</desc>
+        <q xml:lang="gez">ዜና፡ አበው፡ ዘ<placeName ref="INS0101MadhaneAlam"> ቅዱስ፡ መድኃኔ፡ ዓለም።</placeName></q>
+    </item>
+
+  <item xml:id="a2">
+  <locus target="#103r"/>
+  <desc type="ScribalSignature">The name of the scribe was <persName role="scribe" ref="PRS13463Robel">Robel</persName>.</desc>
+  <q xml:lang="gez">ዘሮቤል፡</q>
+  </item>
+
+  <item xml:id="a3">
+  <locus from="1r" to="1v"/>
+  <desc type="Poem"></desc>
+  <q xml:lang="gez"><seg part="I">ንጽሕ<supplied reason="lost">ፍ</supplied>፡ ድንግልናኪ፡ ማርያም፡ እምበረድ፤ ጽሩየ፡ ከዊኖ፡ ወእሜላት፡ ፅዑደ።</seg></q>
+  </item>
+
+  <item xml:id="a4">
+  <locus from="101r" to="101v"/>
+  <desc type="Poem"></desc>
+  <q xml:lang="gez">ኦባዕል፡ ዘኢትነዲ፡ ንጉሥ፡ ወኢይስማዕ፡ ዝክረ፡ ተጽናስ፡ በአጸደ፡ ጸጋከ፡ አምላካዊ፡ አጸደ፡ ጥዒና፡ አምላካዊ፡ ወእንዘ፡ ምጽዋተከ፡ የሐሥሥ፡ ነዳይ፡
+    ይስሐቅ፡ በሠርቀ፡ ዕለታት፡ ስላስ፡ ሞተ፡ በሥጋ፡ ወኀይወ፡ በነፍስ፤<seg part="I"></seg></q>
+  </item>
+
+<item xml:id="e1">
+<locus target="#214v"/>
+<desc type="MixedNote">Scribbles in Arabic.</desc>
```

Comment:
please, add an empty <q> with xml:lang for Arabic, and add the language to the list of language at the end

Reply:
Very clever!